playwright/masonry/utils/waitForRenderedItems.mjs

```diff
@@ -82,7 +82,7 @@ export default async function waitForRenderedItems(
         const { documentElement } = document;
         if (
           typeof scrollHeight !== 'undefined' &&
-          documentElement?.scrollHeight !== scrollHeight
+          documentElement?.scrollHeight >= scrollHeight
```

Comment:
As previously written, this check is a bit odd — we shouldn't need to care that we're at an exact scrollHeight, just that we've surpassed a minimum